src/components/TileGallery/TileGallery.story.jsx

```diff
@@ -0,0 +1,429 @@
+        afterContent={
+          <OverflowMenu
+            floatingMenu
+            onClick={evt => evt.preventDefault()}
+            style={{ height: '2rem' }}
+          >
+            <OverflowMenuItem
+              itemText="Set as home page"
+              onClick={() => alert('Set as home page')}
+            />
+          </OverflowMenu>
+        }
+      />
+      <TileGalleryItem
+        title="Card title"
+        description="card description"
+        moreInfoLink="https://www.ibm.com/br-pt/cloud/internet-of-things?mhsrc=ibmsearch_a&mhq=iot"
+        mode="grid"
+        thumbnail={<Sunny fill="black" description="Icon" width={50} height={50} />}
+        icon={<Check fill="#40be65" onClick={() => alert('favorite')} />}
+        afterContent={
+          <OverflowMenu
+            floatingMenu
+            onClick={evt => evt.preventDefault()}
+            style={{ height: '2rem' }}
+          >
+            <OverflowMenuItem
+              itemText="Set as home page"
+              onClick={() => alert('Set as home page')}
+            />
+          </OverflowMenu>
+        }
+      />
+    </TileGallerySection>
+    <TileGallerySection title="More">
+      <TileGalleryItem
+        title="Card title"
+        description="card description"
+        moreInfoLink="https://www.ibm.com/br-pt/cloud/internet-of-things?mhsrc=ibmsearch_a&mhq=iot"
+        mode="grid"
+        thumbnail={<Sunny fill="black" description="Icon" width={50} height={50} />}
+        icon={<Check fill="#40be65" onClick={() => alert('favorite')} />}
+        afterContent={
+          <OverflowMenu
+            floatingMenu
+            onClick={evt => evt.preventDefault()}
+            style={{ height: '2rem' }}
+          >
+            <OverflowMenuItem
+              itemText="Set as home page"
+              onClick={() => alert('Set as home page')}
+            />
+          </OverflowMenu>
+        }
+      />
+    </TileGallerySection>
+  </TileGallery>
+);
+
+export const galleryData = [
+  {
+    id: 'id1',
+    sectionTitle: 'Favorites',
+    galleryItems: [
+      {
+        title: 'Dashboard title',
+        description: 'More about your dashboard',
+        icon: <IconStarFav />,
+        afterContent: (
+          <OverflowMenu
+            floatingMenu
+            onClick={evt => evt.preventDefault()}
+            style={{ height: '2rem' }}
+          >
+            <OverflowMenuItem itemText="Edit" onClick={() => alert('Edit')} />
+            <OverflowMenuItem itemText="Share" onClick={() => alert('Share')} />
+            <OverflowMenuItem itemText="Move" onClick={() => alert('Move')} />
+            <OverflowMenuItem itemText="Delete" onClick={() => alert('Delete')} />
+          </OverflowMenu>
+        ),
+        thumbnail: <Activity />,
+        href: 'www.ibm.com',
+      },
+      {
+        title: 'Health',
+        description: 'More about your dashboard',
+        icon: <IconStarFav />,
+        afterContent: (
```

Comment:
This is another good spot of repetition that could be pulled out into it's own component for improved readability